### ISSUE TITLE
Batch/Transaction using the client library

### ISSUE THREAD

**Issue by brunovegreville** [smart-on-fhir/client-py#47]
Hey,

I have been using the client library for the last few days and I find it very convenient to get and create resources for a given ResourceType.

However, I have quite struggled to find a way to do a **batch** (even a transaction actually) [ref](https://www.hl7.org/fhir/http.html#transaction) interaction with the FHIR server using this library.

Such use-case happens for instance when trying to get a _summary_ of the health of a given patient (general informations, conditions, medications...). It becomes rapidly annoying and ineffective to do multiple HTTP requests for each ResourceType to be included, hence the use of the batch interaction.

I have not found an intuitive way to use the client library to do so. I basically wanted to know if I had missed something, or if it is not possible right now.

PS: I have been able to make a batch interaction by constructing a Bundle with several "request" objects and then posting this bundle to the base URL using `FHIRServer.post_json`, which returns a `requests` response, hard to parse.

Thank you very much for your help !

**Comment by p2** [comment]
Constructing a Bundle and using `FHIRServer.post_json` is the right way. You can use the requests objects `json()` method to initialize whatever you got back with `FHIRAbstractResource(ret.json()`.

**Comment by brunovegreville** [comment]
Ok, thank you very much for the confirmation !
